docs/sources/plugins/developing/code-styleguide.md

```diff
@@ -60,7 +60,7 @@ Minimal plugin.json:
 
```

Comment:
@marcusolsson , should this style guide be in the `contribute` folder in the repo? Is it redundant with content in the style guides already there?

Reply:
Writing plugins doesn't necessarily mean you mean to contribute to Grafana. I think it makes sense to keep the plugin style guide on grafana.com/docs.